Bottom: 744f8962aaf3b4e0b439ff3bb97be69bad525f36
Top:    62741c4a3c56289629f1e10d0ec4ca2298d5cb0d
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-13 22:08:59 -0300

Thread safety documentation.

for ChangeLog

	* manual/libdl.texi: New.


---

diff --git a/manual/libdl.texi b/manual/libdl.texi
new file mode 100644
index 0000000..e3fe045
--- /dev/null
+++ b/manual/libdl.texi
@@ -0,0 +1,10 @@
+@c FIXME these are undocumented:
+@c dladdr
+@c dladdr1
+@c dlclose
+@c dlerror
+@c dlinfo
+@c dlmopen
+@c dlopen
+@c dlsym
+@c dlvsym
